### PR TITLE
Feat/view editor

### LINKED ISSUE #32: Configurable Toolbar
The toolbar, for symbols, in MathQuill should be made configurable.

See https://moodle.org/plugins/qtype_shortmath

### PR DESCRIPTION
**Pull request by sushanthko**
Merge view-editor branch into develop. Fixes #32

### CODE REVIEW

File: amd/src/editor-manager.js [hunk 1]
```diff
@@ -18,94 +18,160 @@
...
-
-                $('#' + $.escapeSelector('createTemplates')).click(event => {
-                    event.preventDefault();
-                    window.location.replace(editorPath);
-                });
-            }
-        };
+// import * as Str from "core/str";
+export const initialize = (editorPath, actionPath, pluginSettingsPath) => {
+    const _templateElements = document.querySelectorAll(".edit-template, .delete-template");
```
Comment (revosw) [inline amd/src/editor-manager.js:27]: In hindsight I see that I could've just done a `.forEach()`

File: amd/src/input.js [hunk 2]
```diff
@@ -19,64 +19,62 @@
  * @author     André Storhaug <andr3.storhaug@gmail.com>
  * @copyright  2018 NTNU
  */
-
+import VisualMath from "???";
+import Ajax from "core/ajax";
+import notification from "???";
```
Comment (revosw) [inline amd/src/input.js:24]: How do I even import core modules? Relatively, the core modules exist at "../../../../amd/src/<script>.js", but there has to be a better way?

Comment (revosw) [reply amd/src/input.js:24]: @andstor

Comment (andstor) [reply amd/src/input.js:24]: The core modules can be accessed (imported) by writing 
```js
import xxx from "core/moduleName";
```

So the notifications module is available at `core/notification`.

However, `VisualMath` is local to the `ShortMath` repo, so you would probably need to reference it relatively.

**Review by andstor — APPROVED** [review]

**Pull request merged**